frappe/translations/ru.csv

```diff
@@ -4639,8 +4639,8 @@ Not permitted to view {0},Не разрешено просматривать {0}
 Camera,Камера,
 Invalid filter: {0},Недействительный фильтр: {0},
 Let's Get Started,Давайте начнем,
-Reports & Masters,Отчеты и магистры,
-New {0} {1} added to Dashboard {2},Новый {0} {1} добавлен в Личный кабинет {2},
+Reports & Masters,Отчеты и мастеры,
```

Comment:
Google & translate.com say this change means `Reports and wizards` and not `Reports & Masters`. I'm not sure what we should do now. 

Would be nice if we could get verification from some more Russian speakers 😅.

Reply:
"Reports & Masters" refers to a set of records, consolidated data. Does your translation maintain that context?

Reply:
At the moment, the Russian translation of ERPnext is terrible, the Frappe framework translation looks much better against its background, but even a cursory glance at csv reveals many errors.
I tried to make the most literal translation with the preservation of logic, so I suggested ``` "Отчеты и мастеры" ``` as the translation.
But if you make a translation to reveal the context and without verbatim reference to the English version, then the translation of ``` "Отчеты и сводки" ``` will be a more appropriate option.

With this PR, I foremost tried to fix the logic lost during machine translation and fix errors.
In the future, this will allow the Russian community to start using the system without any problems and make further changes to the translation according to the accumulated user experience.

In the next PR, I will try to fix the errors found and improve the already fixed translation based on user experience of our team.

Reply:
@gavindsouza I think we can go ahead with the changes trusting @Altizo as an end-user knows best how the application should be translated for native language usage and will continue to fine-tune translation as required to maintain the correctness.